grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -964,4 +1028,179 @@ class OccurrenceTagLib {
         log.warn "output = ${output}"
         out << output
     }
+
+    @Value('${dataquality.enabled}')
+    boolean dataQualityEnabled
+
+    def ifDataQualityEnabled = { attrs, body ->
```

Comment:
It seems this adds no value compared to just using `<g:if test="${grailsApplication.config.getProperty('dataquality.enabled')}">` in the GSP pages. GSP already use this pattern and you can understand it without having to dig into a taglib to understand what the code is doing. Ditto for code in other taglib calls - just use Groovy `'if ()`.

Reply:
Disagree with this, the `g:if` requires much more mental bandwidth to parse (`getProperty` also needs the type provided making the call even longer and more difficult to read), whereas `<alatag:ifDataQualityEnabled>` is instantly understandable.

Reply:
Well I disagree too. I might assume its checking a config var but I have to step into the code to know for sure. You could be checking for a global env variable or anything inside that code and I can't tell. I can read the code I included in a single glance - `grailsApplication.config.` is peppered throughout the code, so a developer is used to seeing it.  It may be a bit longer but that is still much quicker and easier to grok than having to open another file.

Reply:
Not worth changing though - so keep as is.